README.md

```diff
@@ -2,9 +2,9 @@
 
 [![Build Status](https://travis-ci.org/hmrc/releaser.svg?branch=master)](https://travis-ci.org/hmrc/releaser) [ ![Download](https://api.bintray.com/packages/hmrc/releases/releaser/images/download.svg) ](https://bintray.com/hmrc/releases/releaser/_latestVersion)
 
-Releases artefacts from release candidates with one command. For use as part of a continuous delivery pipeline in which users can create a release from a development commit and create a tag in Github with one command. This automates numerous manual steps required to release an artefact with the existing [release](https://github.com/hmrc/release) scripts.
+Releases artifacts from release candidates with one command. For use as part of a continuous delivery pipeline in which users can create a release from a development commit and create a tag in Github with one command. This automates numerous manual steps required to release an artifact with the existing [release](https://github.com/hmrc/release) scripts.
```

Comment:
`artefact` is the British spelling, `artifact` is the the American. Please change to the original, it wasn't wrong to begin with. https://en.oxforddictionaries.com/definition/artefact

Reply:
I don't consider it wrong, just unconsistent with de-facto standard terminology used by Apache Ivy or Maven projects: http://ant.apache.org/ivy/history/latest-milestone/ivyfile/artifact.html. 
We can also hava a look here: https://books.google.com/ngrams/graph?content=artefact%2Cartifact&case_insensitive=on&year_start=1970&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t4%3B%2Cartefact%3B%2Cc0%3B%2Cs0%3B%3Bartefact%3B%2Cc0%3B%3BArtefact%3B%2Cc0%3B.t4%3B%2Cartifact%3B%2Cc0%3B%2Cs0%3B%3Bartifact%3B%2Cc0%3B%3BArtifact%3B%2Cc0